api/v1/openapi.yaml

```diff
@@ -0,0 +1,953 @@
+          description: Success
+          schema:
+            "$ref": "#/definitions/Configuration"
+        '404':
+          description: Endpoint not found
+    put:
+      summary: Update mutable endpoint configuration
+      description: |
+        Update the configuration of an existing endpoint and regenerates &
+        recompiles the corresponding programs automatically.
+      tags:
+      - endpoint
+      parameters:
+      - "$ref": "#/parameters/endpoint-id"
+      - name: configuration
+        in: body
+        required: true
+        schema:
+          "$ref": "#/definitions/ConfigurationMap"
+      responses:
+        '200':
+          description: Success
+        '400':
+          description: Invalid configuration request
+          x-go-name: Invalid
+        '404':
+          description: Endpoint not found
+        '500':
+          description: Update failed. Details in message.
+          x-go-name: Failed
+          schema:
+            "$ref": "#/definitions/Error"
+  "/endpoint/{id}/labels":
+    get:
+      summary: Retrieves the list of labels associated with an endpoint.
+      tags:
+      - endpoint
+      parameters:
+      - "$ref": "#/parameters/endpoint-id"
+      responses:
+        '200':
+          description: Success
+          schema:
+            "$ref": "#/definitions/LabelConfiguration"
+        '404':
+          description: Endpoint not found
+    put:
+      summary: Modify label configuration of an endpoint
+      description: |
+        Updates the list of labels associated with an endpoint by applying
+        a label modificator structure to the configuration.
+      tags:
+      - endpoint
+      parameters:
+      - "$ref": "#/parameters/endpoint-id"
+      - name: configuration
+        in: body
+        required: true
+        schema:
+          "$ref": "#/definitions/LabelConfigurationModifier"
+      responses:
+        '200':
+          description: Success
+        '404':
+          description: Endpoint not found
+        '460':
+          description: Label to be deleted not found
+          x-go-name: LabelNotFound
+          schema:
+            "$ref": "#/definitions/Error"
+        '500':
+          description: Error while updating labels
+          x-go-name: UpdateFailed
+          schema:
+            "$ref": "#/definitions/Error"
+  "/identity":
+    get:
+      summary: Retrieve identity
+      tags:
+      - policy
+      parameters:
+      - "$ref": "#/parameters/identity-id"
+      - "$ref": "#/parameters/identity-hash"
+      responses:
+        '200':
+          description: Success
+          schema:
+            "$ref": "#/definitions/Identity"
+        '404':
+          description: Identity not found
+        '520':
+          description: Identity storage unreachable. Likely a network problem.
+          x-go-name: Unreachable
+          schema:
+            "$ref": "#/definitions/Error"
+        '521':
+          description: Invalid identity format in storage
```

Comment:
It would likely imply that Cilium was upgraded and the storage has a different schema version which is why I wanted to have a distinct error condition for this.